static/src/js/app/App.vue

```diff
@@ -1,19 +1,14 @@
 <template>
   <div class="app-container">
-    <skeleton>
-      <div slot="left">The left sidebar</div>
-      <div slot="body">The main body of content</div>
-      <div slot="right">The right sidebar</div>
-    </skeleton>
+    <div class="container">
```

Comment:
Does it need the nested `container`?

Reply:
possibly not in the App.vue itself but needs it for margins generally.

Not going to hold up the PR for it but we can revisit once we have more actual "descendant" projects.